stock_lot_production_date/models/product_template.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    use_production_date = fields.Boolean(
+        string="Production Date",
+        help=(
+            "When this box is ticked, the date of production "
+            "will be set automatically on lot/serial numbers."
```

Comment:
AFAICS nothing is going to set the production date automatically in this module. IMO as the module depends only on `stock` it could either provide the field `production_date` on the lot without anything else, or make the field `production_date` required for lots of such products that would be received from a supplier. (but then it would need to be displayed on the stock move line when the picking type has `use_create_lots` set to True, and it needs to be "propagated" when the `stock.production.lot` is created)

Reply:
Yes, I forgot to update this tooltip now we want this module to display only this new field.
Regarding the reception workflow to invite the user to set this production date could be an improvement.

Reply:
I removed the option on `product.template` to get only the field available on the lot.